src/pages/DetailItem/DetailItem.scss

```
@@ -0,0 +1,114 @@
+.container {
+  width: 100vw;
+
+  .topContainer {
+    display: flex;
+    flex-direction: row;
+    margin-top: 100px;
+    border-top: 1px solid #333333;
+
+    .imgBox {
+      width: 65%;
+      display: flex;
+      justify-content: right;
+
+      img {
+        height: 500px;
+        margin-top: 100px;
+      }
+    }
+    .descriptionBox {
+      width: 35%;
+      margin-top: 30px;
+
+      .itemContainer {
+        width: 80%;
+      }
+
+      .titleBox {
+        height: 130px;
+
+        border-bottom: 1px solid #333333;
+      }
+      .title {
+        font-size: 30px;
+      }
+      .description {
+        padding-bottom: 20px;
+      }
+      .categories {
+        .category {
+          border-bottom: 0.5px solid #333333;
+          height: 50px;
```

```suggestion
      .title {
        font-size: 30px;
      }
      
      .description {
        padding-bottom: 20px;
      }
      
      .categories {
        .category {
          border-bottom: 0.5px solid #333333;
          height: 50px;
```
선택자끼리는 가독성을 위해 한칸씩 띄워주세요!

---

수정했습니다!